lib/widgets/draggable_scrollable_modal_bottom_sheet.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/material.dart';
+
+class _DraggableScrollableLayer extends StatelessWidget {
+  const _DraggableScrollableLayer({required this.builder});
+
+  final WidgetBuilder builder;
+
+  @override
+  Widget build(BuildContext context) {
+    return DraggableScrollableSheet(
+        // Enough less than 1 as to not extend into the device's top inset
+        maxChildSize: 0.75,
+
+        // (Things break with `expand: true`, the default; not sure why.)
+        expand: false,
```

Comment:
Huh, weird. Specifically, what I see is that the bottom sheet then starts out occupying the whole screen, as if `initialChildSize` was 1.0 instead of 0.5.

That doesn't seem like what the docs call for. May be a bug. Or maybe it's somehow related to the `Stack` that sits above it?

Reply:
Noting in a code comment.